src/main/resources/application.properties

```diff
@@ -8,4 +8,4 @@ spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.messages.basename=messages
-log.file.output=application.log
+log.file.output=application.log
```

Comment:
```suggestion
log.file.output=application.log

```

Reply:
Can we also add  `spring.jpa.show-sql=true` to show jpa query on logs? What do you think about it?

Reply:
Done! I added it to the application.properties from test (In the prod case, it should be logged with some logging tool {e.g. logback})